arviz/data/io_pymc3.py

```diff
@@ -288,6 +288,8 @@ def constant_data_to_xarray(self):
         for var in self.model.deterministics:
             ancestors = self.theano.tensor.gof.graph.ancestors(var.owner.inputs)
             # no dependency on a random variable
+
+            # pylint: disable=E1101
```

Comment:
I think pylint in [CI builds](https://dev.azure.com/ArviZ/ArviZ/_build/results?buildId=1859&view=logs&j=8b1f4e88-90b9-5fe3-64d9-9b2a3a22d1e4&t=8729fc61-5b5c-512c-a278-229e5112b707&l=20) did not complain about this, it should not be necessary. If it is triggered locally, maybe updating pymc3 and pylint will solve the issue.